SetupCrossTenantRelationshipForTargetTenant.ps1

```diff
@@ -790,7 +833,6 @@ function PreValidation() {
     if ($choice -ne 0) {
         Exit}
     else {Verification}
-
 }
 

```

Comment:
Can everyone validate that I didn't miss something. In my testing after the migrationendpoint was created, it went straight to PreValidation and started prompting for logins. I was skipping the Verification function by changing line 835 to else {Main}

Reply:
why? main is run after prevalidation on line 877

Reply:
? The script has been like this for a while. Do you want me to remove the else and just change it to PreValidation, Verification, Main?

Reply:
honestly have no preference..